config/locales/fr.yml

```diff
@@ -93,6 +110,9 @@ fr:
     follow:
       body: "%{name} vous suit !"
       subject: "%{name} vous suit"
+    follow_request:
+      body: "%{name} a demandé à vous suivre"
+      subject: 'Abonné⋅es en attente : %{name}'
```

Comment:
`attente&nbsp;:`?
If yes, you can make it available on each ` :` in that file :v: